### ISSUE TITLE
AttributeError: module 'dash.dash' has no attribute 'dependencies'

### ISSUE THREAD

**Issue by mokshaxkrodha** [plotly/dash#280]
Below is my error,

```
 Traceback (most recent call last):
  File "app.py", line 54, in <module>
    dash.dependencies.Output('react-graph','figure'),
AttributeError: module 'dash.dash' has no attribute 'dependencies'
```

Below are my imports,

```
from dash import dash
import dash_core_components as dcc
import dash_html_components as html
from dash.dependencies import Input, Output
import plotly.graph_objs as go
import psycopg2
import os
import flask
```

Below is my callback function

`@app.callback(
    dash.dependencies.Output('react-graph','figure'),
    [dash.dependencies.Input('reg_col','value')]
)`

I do not have any file named dash.py in my current directory. I have also tried to change to the import dash from dash to just import dash. The former gives me the below error.
```
> Dash was not successfully imported. Make sure you don't have a file named
'dash.py' in your current directory.
```

**Comment by ned2** [comment]
`from dash import dash` is not right. you want to use *one* of the following:

```
import dash
...
app = dash.Dash()
```
or 

```
from dash import Dash
...
app = Dash()
```

Although your subsequent error message makes me wonder if something else is going on. Does the following minimal Dash example work for you? 

```
import dash
import dash_core_components as dcc
import dash_html_components as html
from dash.dependencies import Input, State, Output

app = dash.Dash()
app.layout = html.Div([
    html.Div(id='target'),
    dcc.Input(id='input', type='text', value=''),
    html.Button(id='submit', n_clicks=0, children='Save')
])


@app.callback(Output('target', 'children'), [Input('submit', 'n_clicks')],
              [State('input', 'value')])
def callback(n_clicks, state):
    return "callback received value: {}".format(state)


if __name__ == '__main__':
    app.run_server(debug=True)
```

**Comment by mokshaxkrodha** [comment]
Couple of updates, 

I did use import Dash but it still gives me the error 'Dash was not successfully imported. Make sure you don't have a file named 'dash.py' in your current directory.' 

Likewise, its possible something is wrong with my PYTHONPATH probably, so heres the output for my import dash ; print(dash.__file__) -- C:\Users\nilay.doshi\AppData\Local\Continuum\anaconda3\lib\site -packages\dash_init_.py

**Comment by ned2** [comment]
It sounds like something is wrong with the installation of the Dash package. What does the contents of that `__init__.py` look like? It should be this:

https://github.com/plotly/dash/blob/master/dash/__init__.py

**Comment by mokshaxkrodha** [comment]
Thats my __init__.py file had some garbage which I changed to the contents from the link. Its got it working.

**Comment by ned2** [comment]
If your problem is resolved would you be able to close this issue? Thanks :)

**Comment by mokshaxkrodha** [comment]
I would assume the same too but I guess there was some mix up since the contents of you `__init__.py` were  not the same. But yes its resolved.

**Comment by JiriHamr** [comment]
Hey guys, I have the same problem as @mokshaxkrodha had. I was using Dash yesterday and everything was alright, but today the problem occured from nowhere.

I followed your instructions and the problem is not resolved. I also had a different content of my 
`__init__.py`. Unfortunately, @ned2  your suggestion didn't help.

Do you have any ideas what could be another cause of this problem?

Thanks!
.

**Comment by JiriHamr** [comment]
Hi again, reinstalling the whole Dash via pip resolved the problem for me.

Cheers

**Comment by Vondoe79** [comment]
Hi everyone, 
I know this issue is an old one. But I just ran into it as well so, I figured I add mine to this thread for any help I could get. 

@ned2 \ I have a similar issue. And I have checked the `__init__.py` file. Every line of code looks exactly as in the referenced `__init__.py` file above - https://github.com/plotly/dash/blob/master/dash/__init__.py. But the issue still persists.  

 @JiriHamr \ I have also tried uninstalling and installing, but it did not fix the issue. 

 I am using Python version `3.9.13`, and `dash 2.14.1`. I have tried everything, but the error is not going away.
